lib/generator/stubs/remote_data_source_stub.dart

```diff
@@ -1,7 +1,8 @@
 String remoteDataSourceStub({required String name}) => '''
+import 'package:fixit/core/data_soruce.dart';
 import 'package:fixit/core/api_client.dart';
 
-class ${name}RemoteDataSource {
+class ${name}RemoteDataSource extends DataSource{
```

Comment:
add space after between DataSource and { symbol  
`class ${name}RemoteDataSource extends DataSource {`

Reply:
solved